### PR TITLE
Lists

### PR DESCRIPTION
**Pull request by uday-nv-work**
Added lists component

### CODE REVIEW

**Review by srishtigautam21 — APPROVED** [review]
Clean code
good to merge

File: component/Lists/lists.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	margin: 3rem auto;
+	max-width: 35rem;
+	padding: 2rem 0;
+	list-style: none;
+}
+.stacked-list li {
+	background: var(--offgreen);
+	border-bottom: medium solid var(--COLOR-BRAND);
+	border-radius: 0.2rem;
+	font-size: 2rem;
```
Comment (bhushannaikaroskar) [inline component/Lists/lists.css:42]: you can use px values here as value seem too small

File: component/Lists/lists.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+.list-disc {
+	list-style: disc;
+}
+.list-circle {
+	list-style: circle;
+}
+.list-square {
+	list-style: square;
+}
+.list-none {
```
Comment (bhushannaikaroskar) [inline component/Lists/lists.css:22]: please make proper spacing between two class declarations for easy readability

**Review by bhushannaikaroskar — APPROVED** [review]
Hey Uday, good can clean code. Liked the way you have separated list styles for different types. I have given some  suggestions please check before merging. Approving the merge provided that you make these small changes.

**Review by zoyatanweer — APPROVED** [review]
- clean code, easy to understand
- do the suggested changes and you're good to merge
- good work

**Pull request merged**